com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -377,65 +384,42 @@ public static void FillDataLocation(ref DataLocation loc, SphericalHarmonicsL2[]
                             // We are processing chunks at a time.
                             // So in practice we can go over the number of SH we have in the input list.
                             // We fill with black to avoid copying garbage in the final atlas.
```

Comment:
Comment will need updating a bit to make it more clear that is not black, but the value that will result in black after decoding